Resources/services/services.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+
+
+    <services>
+        <service class="MollieShopware\Services\MollieOrderRequestAnonymizer\MollieOrderRequestAnonymizer" id="mollie_shopware.services.mollie_order_request_anonymizer.mollie_order_request_anonymizer" public="true">
```

Comment:
could you improve the id please
seems very generated
try a simple "mollie_shopware.services.order_request_anonymizer",
thats totally fine
because of interfaces and options to have separate "beautiful" ids instead of shopware6, i try to create a good-named DI setup where i could easily switch implementation classes :)